### DISCUSSION TITLE
Extended suggestion length

### DISCUSSION THREAD

**Discussion by bannsec** [community/community#6661] [Copilot Conversations]
I've found that copilot will truncate it's suggestion lengths. I can accept a truncated suggestion and hit tab, hit enter, hit tab, etc until it's finally filled in. It would be helpful to have a full "accept a longer suggestion" option or something.

**Comment by orph** [comment]
@bannsec thanks for the report - can you post a video of what you're seeing so we can tell if Copilot is operating as expected?

**Reply by bannsec** [reply]
Attached is an example video. The 'wait-tab-enter' loop went on for a bit longer, but that's the idea.


https://user-images.githubusercontent.com/7632864/138734730-8c1aa90e-ced7-49a0-90bd-2718350dc997.mp4
